projects/sandbox/src/app/dataset-content-summary/dataset-content-summary.component.html

```diff
@@ -17,38 +17,56 @@
   </ul>
 </ul>
 
-<div class="tier-data-grid">
+<div
+  *ngIf="gridData"
+  class="tier-data-grid"
+  [ngClass]="{
+    'sort-asc': sortDirection === 1,
```

Comment:
my suggestion would be to use a constant to indicate sortDirection. ASC=1, DESC=-1, NEUTRAL=0

Reply:
Done